x/stakeibc/keeper/reward_converter.go

```diff
@@ -254,9 +254,9 @@ func (k Keeper) SwapRewardTokens(ctx sdk.Context, rewardAmount sdkmath.Int, rout
 		"Preparing MsgSwapExactAmountIn of %+v from the trade account", msg.TokenIn))
 
 	// Timeout the swap at the end of the epoch
-	strideEpochTracker, found := k.GetEpochTracker(ctx, epochstypes.STRIDE_EPOCH)
+	strideEpochTracker, found := k.GetEpochTracker(ctx, epochstypes.HOUR_EPOCH)
```

Comment:
Good catch, I wasn't even thinking about that the faster call cadence to the swap would cause these timeouts to overlap unless we used the hour cadence.